src/background_workers.c

```diff
@@ -391,19 +391,16 @@ void *RedisAI_Run_ThreadMain(void *arg) {
 
                 // Run is over, now iterate over the run info structs in the batch
                 // and see if any error was generated
-                int dagError = 0;
+                int first_dag_error = 0;
```

Comment:
Is there no usage for `dagError` after this block?
also
please use `bool` or dedicated `enum` instead of `int`

Reply:
no usage... 
Changed to bool